apps/analysis_framework/schema.py

```diff
@@ -158,10 +158,9 @@ def resolve_used_in_project_count(root, info):
         return info.context.dl.analysis_framework.af_project_count.load(root.id)
 
 
-class AnalysisFrameworkRoleType(DjangoObjectType):
+class AnalysisFrameworkRolesListType(DjangoObjectType):
```

Comment:
Type name shouldn't be plural
```suggestion
class AnalysisFrameworkRoleListType(DjangoObjectType):
```